runtime/compiler/x/runtime/X86RelocationTarget.cpp

```diff
@@ -120,7 +120,7 @@ j9ThunkInvokeExactHelperFromTerseSignature(UDATA signatureLength, char *signatur
          break;
       case '[':
          /* intentional fall-through */
-      case 'L':
+      case 'L': //do we have a corresponding helper for Q?
```

Comment:
I would suggest you add a case for `Q` that falls through to the handling for `L` for now, but add a  TODO that investigation is needed.